index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Signup Form</title>
+    <meta name="description" content="Sign up for an intuit account">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+
+
+    <div id="sign-in">
+        <p>Already have an Intuit account?<br>
+            I forgot my User ID or Password Sign In</p>
+        <button>Sign in</button>
+    </div>
+
+    <header>
+        <img alt="Intuit logo" id="intuit-logo" src="assets/logo-intuit.png">
+
+        <div id="small-logos">
+            <img alt="Mint logo" src="assets/logo-mint.png">
+            <img alt="quickbooks logo" src="assets/logo-quickbooks.png">
+            <img alt="turbotax logo" src="assets/logo-turbotax.png">
+
+        </div>
+    </header>
+
+    <div class="frame">
```

Comment:
- [OPTIONAL] Consider the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document